src/client/components/SignUpForm/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './index.scss';
+
+const SignUpForm = (props) => (
+  <form className="signup-form" onSubmit={props.onSubmit}>
+    <input name="username" style={props.style.userName} required placeholder="Username" />
```

Comment:
be consequent, use dash in class names or camelcase but dont mix them. Discuss it with your team